content/posts/2023-06-17-what-does-object-assign-do.mdx

```diff
@@ -12,28 +12,28 @@ This one is easy. It just merges properties of one object with the properties of
 Let's look at an example:
 
 ```typescript
-const person = {
-  name: "John",
-  age: 30,
-  gender: "male",
-};
-
-const horse = {
-  name: "Spirit",
-  age: 20,
-  color: "brown",
-};
-
-const centaur = Object.assign(person, horse);
-console.log('centaur:', centaur)
-/*
-  {
-    name: 'Spirit',
-    age: 20,
-    gender: 'male',
-    color: 'brown'
-  }
-*/
+const person = {
+  name: "John",
+  age: 30,
+  gender: "male",
+};
+
+const horse = {
+  name: "Spirit",
+  age: 20,
+  color: "brown",
+};
+
+const centaur = Object.assign(person, horse);
+console.log('centaur:', centaur)
+/*
+  {
+    name: 'Spirit',
+    age: 20,
+    gender: 'male',
+    color: 'brown'
+  }
+*/
```

Comment:
Why this file has been changed?

Reply:
I don't know, really🤔 I didn't change this file at all